marketplace/deployer_util/config_helper_test.py

```diff
@@ -514,6 +520,116 @@ def test_unknown_type(self):
                   type: UNKNOWN
             """))
 
+  def test_v2_fields(self):
```

Comment:
nit: there are multiple asserts here, I'm usually a fan of 1:1 mapping of test case and asserts. My suggestion would be to have the load_yaml call in a setup function and have one test case per assert

Reply:
All of these fields are required in v2, so we have to specify all of them anyway.